config/index.js

```diff
@@ -17,16 +17,24 @@ const DEFAULT_TIMEOUT = 2000;
 
 const overridableFields = [
 	'tokenKey', 'tokenPassword', 'testPackId', 'concurrency', // launcher automated
-	'username', 'password', 'orgId', 'deviceId', 'appConfigId', 'inspect', 'inspectBrk', // launcher intaractive
-	'logLevel', 'logDir', 'timestamp', 'configFile', 'disallowCrashReports', 'defaultTimeout', // launcher common
+	'username', 'password', 'orgId', 'deviceId', 'appConfigId', 'inspect', 'inspectBrk', // launcher interactive
+	'logLevel', 'logLevelTestLines', 'logLevelTestErrors', 'logLevelNetworkLogs', 'logLevelConsoleLogs', // launcher common
+	'logLevelDebug', 'logLevelTestLauncher', 'logDir', 'timestamp', 'configFile', 'disallowCrashReports', // launcher common
+	'defaultTimeout', // launcher common
 ];
 
-const configurableFields = ['logLevel', 'disallowCrashReports', 'defaultTimeout'];
+const configurableFields = ['logLevel', 'logLevelTestLines', 'logLevelTestErrors', 'logLevelNetworkLogs',
+	'logLevelConsoleLogs', 'disallowCrashReports', 'defaultTimeout'];
 
 const main = Object.freeze({
 	apiUrl: 'https://the.suite.st/api/public/v3',
 	disallowCrashReports: false,
-	logLevel: logLevels.normal,
+	logLevelTestLines: logLevels.normal,
+	logLevelTestErrors: logLevels.normal,
+	logLevelNetworkLogs: logLevels.normal,
+	logLevelConsoleLogs: logLevels.normal,
+	logLevelTestLauncher: logLevels.normal,
```

Comment:
@andriichumak that's not in ticket but I added as we have some logs that doe's not relate to any of type above and with 'logLevel': 'silent' they should not be logged